catkin_ws/src/vision/src/object_detection_utils.py

```diff
@@ -70,6 +84,33 @@ def getPointCloud(self, bbox=None):
             return self.cleanPointCloud(self.point_cloud)
         else:
             return self.cleanPointCloud(cropToBbox(self.point_cloud, bbox, copy=True))
+    def updateRGB(self, msg):
+        temp = bridge.imgmsg_to_cv2(msg)
+        self.rgb = temp/255
+        if self.depth is not None:
+            self.updatePointCloud()
+    def updateDepth(self, msg):
+        temp = bridge.imgmsg_to_cv2(msg)
+        self.depth = temp/depth_scale_factor
+        self.updatePointCloud()
+    def updateCameraInfo(self, msg):
+        if(self.y_over_z_map is not None): return
+        fx = msg.K[0]
+        fy = msg.K[4]
+        cy = msg.K[2]
+        cx = msg.K[5]
+        self.width = msg.width
+        self.height = msg.height
+
+        u_map = np.tile(np.arange(self.width),(self.height,1)) +1
+        v_map = np.tile(np.arange(self.height),(self.width,1)).T +1
+
+        self.x_over_z_map = (cx - u_map) / fx
+        self.y_over_z_map = (cy - v_map) / fy
+        if self.depth is not None:
+            self.updatePointCloud()
+        else:
+            print("Yo")
```

Comment:
lmao what does "Yo" mean?

Reply:
Ah sorry about that! Forgot to remove after testing haha